requirements-dev.txt

```diff
@@ -34,4 +35,4 @@ flair-visual>=0.5.3; python_version >= "3.10"
 sympy
 pytest
 pytest-cov
-packaging
\ No newline at end of file
+packaging
```

Comment:
Why is `packaging` required?

Reply:
I don't remember adding this manually? removed.